### DISCUSSION TITLE
Unmount folder after build

### DISCUSSION THREAD

**Discussion by narian** [dokku/dokku#4900] [Q&A]
Hello!
I'm trying to build my nodejs project with mounted directory that has been connected through docker-options, but always stuck on error (resource busy) when herokuish is trying to delete mounted folder. Is there any hook or command for ignoring this folder or for unmounting it before deletion? Thanks.

**Comment by josegonzalez** [comment]
We don't directly support file mounts during the build process, and there is no way to disable that deletion. Is there a reason you need to mount a directory at that time?

Might be worth mounting at a path that isn't inside `/app`, as then herokuish shouldn't touch it (in theory).
